saul-core/src/main/scala/edu/illinois/cs/cogcomp/saul/classifier/infer/InferenceCondition.scala

```diff
@@ -33,5 +29,5 @@ abstract class InferenceCondition[INPUT <: AnyRef, HEAD <: AnyRef](solver: ILPSo
   }
 
   val outer = this
-  def convertToType[T <: AnyRef](implicit tag: ClassTag[T]): InferenceCondition[T, HEAD] = this.asInstanceOf[InferenceCondition[T, HEAD]]
+  def convertToType[T <: AnyRef]: InferenceCondition[T, HEAD] = this.asInstanceOf[InferenceCondition[T, HEAD]]
```

Comment:
The `tag` is not used anywhere. Currently only `Node` and `Property` subclasses have tags that are being used.